### ISSUE TITLE
colmap2mvsnet.py issue

### ISSUE THREAD

**Issue by mxgbs** [jzhangbs/Vis-MVSNet#39]
Got error, the invalid key in the end. How could this work since if you run to the end, i+1 will out of range?? Please advice. Thank u

 for i in range(num_images):
        zs = []
        for p3d_id in images[i+1].point3D_ids:

**Comment by jzhangbs** [comment]
Hi @mxgbs, `images` is a dict with image id starting from 1 instead of 0, you can see all the indexing are with `+1`. Maybe some of the images are discarded in sfm. A simple test is to try other scans.

**Comment by flow-specter** [comment]
If this situation occurs—'Maybe some of the images are discarded in SfM'—how should it be handled?

**Comment by jzhangbs** [comment]
maybe the simplest solution is to remove the missing images and run colmap again

**Comment by flow-specter** [comment]
Yes, this would be simpler, but it would lead to unnecessary time costs.
